### PR TITLE
Replacing boilerplate Vite with jsPsych + nivturk + autora-user-cookiecutter content

### PR DESCRIPTION
**Pull request by krachwal**
`npm run dev` loads the consent form via jsPsych timeline.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Web based behavioral closed loop
```
Comment (krachwal) [inline README.md:1]: This is their current README. I removed references to Firebase but we will need to expand it further when we have the project more fleshed out.

File: experiment/.eslintignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (krachwal) [inline experiment/.eslintignore:1]: Added eslint to autoformat in VS Code. No need to check formatting in 3rd party dependencies or final minified code.

File: experiment/components/consent.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
+<!doctype html>
```
Comment (krachwal) [inline experiment/components/consent.html:1]: generic copy of the consent form from nivturk which felt robust and was sparse in autora-user-cookiecutter

File: experiment/index.html [hunk 4]
```diff
@@ -2,12 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
```
Comment (krachwal) [inline experiment/index.html:4]: replacing boilerplate Vite html

File: experiment/main.js [hunk 5]
```diff
@@ -1,24 +1,40 @@
-import './style.css'
-import javascriptLogo from './javascript.svg'
-import viteLogo from '/vite.svg'
-import { setupCounter } from './counter.js'
+import { initJsPsych } from "jspsych";
+import jsPsychExternalHtml from '@jspsych/plugin-external-html';
+import "./style.css";
```
Comment (krachwal) [inline experiment/main.js:4]: replacing boilerplate Vite code with jsPsych timeline that loads the consent form

File: experiment/style.css [hunk 6]
```diff
@@ -1,96 +1,143 @@
 :root {
```
Comment (krachwal) [inline experiment/style.css:1]: css mashup from boilerplate and nivturk and autora-user-cookiecutter

File: experiment/vite.config.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+import eslint from 'vite-plugin-eslint';
+// for the following see https://github.com/cypress-io/cypress/issues/25397#issuecomment-1775454875
+import dns from 'dns';
+dns.setDefaultResultOrder('ipv4first');
+
+export default defineConfig(() => {
+	return {
+		build: {
+			outDir: 'dist'
+		},
```
Comment (krachwal) [inline experiment/vite.config.js:11]: setting build target and base server path

**Review by varun646 — APPROVED** [review]

**Pull request merged**